src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -33,14 +22,935 @@ to make sure IE does not go into compatibility mode.
+
+For more information [have a look at the scopes and hub documentation]({%- link
+_documentation/enriching-error-data/scopes.md -%}).
+
+&nbsp;
+### Capturing the User
+Sending users to Sentry will unlock many features, primarily the ability to drill down into the number of users affecting an issue, as well as to get a broader sense about the quality of the application.
+
+Capturing the user is fairly straight forward:
+
+```
+Sentry.configureScope((scope) => {
+  scope.setUser({"email": "john,doe@example.com"});
+});
+```
+
+Users consist of a few critical pieces of information which are used to construct a unique identity in Sentry. Each of these is optional, but one **must** be present for the Sentry SDK to capture the user:
+
+`id`
+
+: Your internal identifier for the user.
+
+`username`
+
+: The user's username. Generally used as a better label than the internal ID.
+
+`email`
+
+: An alternative, or addition, to a username. Sentry is aware of email addresses and can show things like Gravatars, unlock messaging capabilities, and more.
+
+`ip_address`
+
+: The IP address of the user. If the user is unauthenticated providing the IP address will suggest that this is unique to that IP. If available, we will attempt to pull this from the HTTP request data.
+
+Additionally, you can provide arbitrary key/value pairs beyond the reserved names, and the Sentry SDK will store those with the user.
+
+&nbsp;
+### Tagging Events
+Sentry implements a system it calls tags. Tags are various key/value pairs that get assigned to an event, and the user can later use them as a breakdown or quick access to finding related events.
+
+Most SDKs generally support configuring tags by configuring the scope:
+
+```
+Sentry.configureScope((scope) => {
+  scope.setTag("page_locale", "de-at");
+});
+```
+
+Several common uses for tags include:
+
+-   The hostname of the server
+-   The version of your platform (e.g. iOS 5.0)
+-   The user’s language
+
+Once you’ve started sending tagged data, you’ll see it show up in a few places:
+
+-   The filters within the sidebar on the project stream page.
+-   Summarized within an event on the sidebar.
+-   The tags page on an aggregated event.
+
+We’ll automatically index all tags for an event, as well as the frequency and the last time the Sentry SDK has seen a value. Even more so, we keep track of the number of distinct tags and can assist you in determining hotspots for various issues.
+
+&nbsp;
+### Setting the Level
+You can set the severity of an event to one of five values: `fatal`, `error`, `warning`, `info`, and `debug`. `error` is the default, `fatal` is the most severe and `debug` is the least severe.
+
+**[Mimi note: Is there a way to set the level in a manually triggered error? This way, the level isn't set in scope -- which could be overly complicated/messy.]**
+
+**[Mimi note: vv this example doesn't actually set a level?? It's what I found under Context > Setting the Level]**
```

Comment:
@kamilogorek am I reading this right? Looks like a scopes thing and not actually setting the level.